agent/lib/kontena/service_pods/migrator.rb

```diff
@@ -0,0 +1,23 @@
+module Kontena
+  module ServicePods
+    class Migrator
+
+      # @param service_container [Docker::Container]
+      # @return [Docker::Container]
+      def self.migrate_container(service_container)
+        service_container.update({
+          'RestartPolicy' => {}
```

Comment:
It would be nicer to have some kind of explicit `migrate_restart_policy` method with a comment saying that version 1.4 etc - these are things we're going to need to keep track of in the future as well.

Should also get logged.